### DISCUSSION TITLE
Why is there data in 'raw', but not in 'validated_data'?

### DISCUSSION THREAD

**Discussion by ashishkumardas1025** [eyurtsev/kor#264] [General]
This is error I am facing extracted data is not stored in data rather stored in raw is it regarding that the KOR is not stable?
{'data': {},
 'raw': '<json>[\n{\n"Project Name": ["GoWireless"],\n"Start Date and End Date": [],\n"Role": ["Tech Lead/ Architect"],\n"Technologies": ["Microsoft Azure", "Microsoft Power BI"],"Description": ["Responsible for developing EL jobs via Talend ETL tool and loading into the future database. Analyzed data requirements from a business perspective. Converted business needs into technical details."],\n"Responsibility": ["Developed EL jobs via Talend ETL tool", "Analyzed data requirements", "Converted business needs into technical details", "Developed and maintained MYSQL banking database in AWS cloud"]\n}\n]</json>',
 'errors': [kor.exceptions.ParseError('The LLM has returned structured data which does not match the expected schema. Providing additional examples may help improve the parse.')],
 'validated_data': {}}

**Comment by eyurtsev** [comment]
This is not a Kor issue, but an issue with the output from the LLM itself.

```python
'errors': [kor.exceptions.ParseError('The LLM has returned structured data which does not match the expected schema. Providing additional examples may help improve the parse.')
```

The data does not match the pydantic schema. See guidelines on how to improve the quality of results when working with LLMs.
